### ISSUE TITLE
Restrict players fro picking up items spawned in by staff members of a server

### ISSUE THREAD

**Issue by Aceheliflyer** [uEssentials/uEssentials#444]
I have a case of Moderators and Administrators giving items to players on my server when I really want my server to be a survival server. I still want my staff members to be able to spawn items in for themselves, and for them only. Is it possible to restrict staff members of a server giving players items just by dropping it?

**Comment by WGSRangerWolf** [comment]
Yes. this is only done with permissions not Unturned administrator stuff
/admin uses the unturned's admin system
but if you were to make an administrator rank you would have to do something like this in your permissions.xml and to add them you do /p add <name> <group id>
```
<Group>
      <Id>mod</Id>
      <DisplayName>Moderator</DisplayName>
      <Color>yellow</Color>
      <ParentGroup>default</ParentGroup>
      <Permissions>
        <Permission Cooldown="0">kick</Permission>
        <Permission Cooldown="0">ban</Permission>
      </Permissions>
    </Group>
```

**Comment by Aceheliflyer** [comment]
That's actually what I use, I was just wondering if there is someway to keep administrators from dropping items on the ground for guests to pick up

**Comment by WGSRangerWolf** [comment]
Hmm at this moment i do not but ill forward this to @leonardosnt
